### PR TITLE
Committing this to get discussion going.

### PR DESCRIPTION
**Pull request by adamdavis40208**
What do we do if we start seeing json.level:ERROR without the json fault string (maybe a loggly issue, maybe a bug in gtwy, etc)

### CODE REVIEW

File: lib/lita/handlers/onewheel_loggly.rb [hunk 1]
```diff
@@ -351,12 +351,8 @@ def rollup_events(events)
           # Let's see what we've got.
           Lita.logger.debug event
           msg = JSON.parse(event['logmsg'])
-          message = msg['message']
-          # cut out req_url
-          if message.include? 'req_url'
-            start = message.index('req_url=') + 8
-            the_end = message.index(',', start)
-            url = message[start..the_end]
+          if event.key? 'event' and event['event'].key? 'json' and event['event']['json'].key? 'req_url'
```
Comment (adamdavis40208) [inline lib/lita/handlers/onewheel_loggly.rb]: I want to add an error that doesn't have req_url and make sure it doesn't explode

File: lib/lita/handlers/onewheel_loggly.rb [hunk 2]
```diff
@@ -389,60 +385,9 @@ def process_event(events, alerts)
...
-          elsif md = /(requests.exceptions.ReadTimeout.*read timeout=\d+\))/.match(message)
-            salient = "Unhandled #{md[0]}"
-            alerts[salient] += 1
-          else
-            if md = /(x-amzn-requestid=[\w-]+)/.match(message)
-              salient = message.gsub /\\n/, "\n"
-              Lita.logger.debug "UNKNOWN #{salient}"
-              alerts["Unknown #{md[0]}"] = 1
-            end
+          if event.key? 'event' and event['event'].key? 'json' and event['event']['json'].key? 'fault'
```
Comment (adamdavis40208) [inline lib/lita/handlers/onewheel_loggly.rb]: ditto here, what do we do if an error doesn't have a fault code. I guess we could just log a blank error bucket?

**Comment by adamdavis40208** [comment]
So the nice thing about this PR is that it cleans up the code. The bad is that it's only useful for JSON logs (and not both). 

I also made a version that did both, but I just had an itch to delete code :P

**Comment by onewheelskyward** [comment]
Committed via https://github.com/onewheelskyward/lita-onewheel-loggly/commit/b3384955b8c9bdffed2b23d98bd2e2ec6fd9c38e

**Pull request closed without merging**